### DISCUSSION TITLE
Is there a way to remove the blog title from the title of each post?

### DISCUSSION THREAD

**Discussion by talonx** [facebook/docusaurus#10673] [Q&A]
In blog-only mode, each blog posts's title shows up as "post title | blog title". Is there a way to not have the "blog title" part?

**Comment by slorber** [comment]
You can with swizzle, but we aim to make it easier in the future

See also related issues:
- https://github.com/facebook/docusaurus/issues/5878
- https://github.com/facebook/docusaurus/issues/10045

We also have a `blogPost.frontMatter.title_meta` in v3.6: https://github.com/facebook/docusaurus/pull/10586

**Reply by talonx** [reply]
Thank you.
